R/utils.R

```diff
@@ -155,12 +172,45 @@ invokeRemote = function(map, method, args = list()) {
     )
   )
 
+  sess <- map$session
   if (map$deferUntilFlush) {
-    map$session$onFlushed(function() {
-      map$session$sendCustomMessage("leaflet-calls", msg)
-    }, once = TRUE)
+    if (compareVersion(as.character(packageVersion("shiny")), "0.12.0.9001") < 0) {
```

Comment:
You may know this already, but you can just do `packageVersion("shiny") < "0.12.0.9001"`.